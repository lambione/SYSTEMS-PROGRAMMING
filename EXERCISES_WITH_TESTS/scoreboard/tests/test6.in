team3 team2 10 2
team2 team1 1 0
team1 team3 100 101
