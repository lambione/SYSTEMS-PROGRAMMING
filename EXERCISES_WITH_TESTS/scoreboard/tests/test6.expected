team3 6 2
team2 3 2
team1 0 2
